recipe/emailverification/utils.go

```diff
@@ -0,0 +1,30 @@
+package emailverification
+
+import (
+	"github.com/supertokens/supertokens-golang/recipe/emailverification/schema"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func ValidateAndNormaliseUserInput(appInfo supertokens.NormalisedAppinfo, config schema.TypeInput) schema.TypeNormalisedInput {
+	getEmailVerificationURL := *config.GetEmailVerificationURL
+	if config.GetEmailVerificationURL == nil {
+		getEmailVerificationURL = GetEmailVerificationURL(appInfo)
+	}
+	createAndSendCustomEmail := *config.CreateAndSendCustomEmail
+	if config.CreateAndSendCustomEmail == nil {
+		createAndSendCustomEmail = CreateAndSendCustomEmail(appInfo)
+	}
+
+	override := schema.Override{
+		Functions: config.Override.Functions,
```

Comment:
What if the user has not provided the `override` object? The default impl needs to be there..